app/steps/ui/deceased/otherNames/template.html

```diff
@@ -0,0 +1,66 @@
+{% extends "includes/two_thirds_form.html" %}
+
+{% from "widgets/textfields.html" import textField %}
+
+{% block error_summary %}
+    {% if errors %}
+    <div class="error-summary" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
+        <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
+            {{common.errorSummaryHeading | safe}}
+        </h2>
+        <ul class="error-summary-list">
+
+        {% for index, otherNameErros in errors %}
+            {% for error in otherNameErros %}
+                <li><a href="#otherNames_{{index}}_{{error.param}}">{{error.msg.summary | safe}}</a></li>
+            {% endfor %}
+        {% endfor %}
+        </ul>
+    </div>
+    {% endif %}
+{% endblock %}
+
+{% block question %}
+    <h1 class="form-title heading-large">{{content.question | replace('{deceasedName}', fields.deceasedFullName.value) | safe }}</h1>
```

Comment:
As above

Reply:
agreed with David to leave template substitution as PA journey